apps/jobs/admin.py

```diff
@@ -18,10 +18,10 @@ class SubmissionAdmin(ImportExportTimeStampedAdmin):
                     'created_by', 'status', 'is_public', 'submission_number', 'submitted_at',
                     'execution_time', 'input_file', 'stdout_file', 'stderr_file',
                     'submission_result_file', 'submission_metadata_file',)
-    list_filter = ('participant_team', 'challenge_phase',
+    list_filter = ('challenge_phase__challenge', 'participant_team', 'challenge_phase',
```

Comment:
@vkartik97 Thanks for the PR. Can we please remove `participant_team` from here. Although, it wasn't mentioned in the deliverables.